runtime/onert/backend/cpu/ConstantInitializer.h

```diff
@@ -35,6 +35,15 @@ class ConstantInitializer : public IConstantInitializer
   ConstantInitializer(const ir::Operands &operands,
                       const std::shared_ptr<TensorBuilder> &tensor_builder);
 
+public:
+  void registerDefaultInitializer(const ir::OperandIndex &index, const ir::Operand &obj) override;
+
+  // TODO: For now the only cpu backend supports constant tensor to use data from external
+  // If the other backend supports (to do this,
+  // ExternalTensor should be abstract such as IExternal, maybe),
+  // this can be an interface of IConstantInitializer
```

Comment:
IMHO I think we rather find another way of constant initialization since this class was first introduced as a workaround of initialization whether copy or permute-copy.

Reply:
That's the interesting point but seems not easy :-)

Reply:
Yeah, I'm going to try it soon(maybe) as part of my refactoring project.